### ISSUE TITLE
Easy way to move image/use old image 

### ISSUE THREAD

**Issue by Eldair** [microsoft/WSL#8156]
I recently had to reinstall windows and came upon this frustrating design of wsl. There is no way to use old ext4 file. I have to install new ubuntu, export that image in order to change it's location (my previous Ubuntu is very large) then overwrite new ext file with my old file.

PS I don't want my image to be in default location to not lose it when reinstalling windows.

Same thing for docker and their image(s) - data and desktop-data - why can't I set defeault locations during install or use old ones - it's just too much work to set it all up.

**Comment by LuanVSO** [comment]
the ubuntu ext4 vhd is located in
`%AppData%\Local\Packages\CanonicalGroupLimited.Ubuntu*\LocalState\`
you can just overwrite it to be whatever you want after the distro is registered with wsl.
just make sure you run `wsl --shutdown` beforehand

**Comment by Eldair** [comment]
Yes, that is true but if I want to move image to another drive (so it wouldn't be deleted in case of next windows reinstall) I have to export then import again which is cumbersome with huge images, it should be easier.

**Comment by LuanVSO** [comment]
A symbolic link can be used to do that

**Comment by benhillis** [comment]
I've considered adding the ability to export / import VHD's directly instead of as .tar files, would that satisfy your requirements?

**Comment by Eldair** [comment]
That would certainly help a lot - so the "imported" VHD would stay in the same location and I could mark it as default (no need to import - export - import to set location)?

**Comment by benhillis** [comment]
Yes that is what I was thinking, basically a way to just point to some existing VHD.

**Comment by Eldair** [comment]
That would be amazing - much better and faster way to restore backups (and create them by just copying)!
